src/metaBlob.cxx

```diff
@@ -134,12 +134,18 @@ PointDim(void) const
 }
 
 void MetaBlob::
-NPoints(size_t npnt)
+NPoints(int npnt)
 {
   m_NPoints = npnt;
 }
 
-size_t MetaBlob::
+void MetaBlob::
+NPoints(size_t npnt)
```

Comment:
This may not work on a 32-bit system build because size_t and int will have the same size.  Please verify that the different sign is enough to prevent build errors on 32-bit builds.